taskana-adapter-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java

```diff
@@ -140,4 +180,35 @@ public void process_instance_with_multiple_executions_should_result_in_multiple_
             assertEquals(taskanaTaskExternalId, camundaTaskId);
         }
     }
-}
\ No newline at end of file
+
+    private void createTaskanaTaskAndVerifyTaskVariables(String camundaTaskId, String assumedVariablesString) {
+
```

Comment:
please rename this method: the testcase doesn't create a taskana task, it rather relies on the adapter to have already created a taskana task. The testcase simply checks that the taskana task exists and has the correct content ....

Reply:
Done